sdks/python/apache_beam/examples/wordcount_it_test.py

```diff
@@ -103,13 +50,6 @@ def test_wordcount_it(self):
   def test_wordcount_fnapi_it(self):
     self._run_wordcount_it(wordcount.run, experiment='beam_fn_api')
 
-  @attr('Py3IT')
```

Comment:
Do we need to do anything to run wordcount on Python 3 using regular wordcount test?

Reply:
I don't think we need to do anything particular. Setup python3 environment and use python3 interpreter should be sufficient.

Reply:
So, will we continue to run wordcount tests for Python3 after this change is submitted?

Reply:
yes. I just updated the code. Changes in `run_validatescontainer_py3.sh` should be clear. Basically `ValidatesContainer` will be run.